app/controllers/catalog_controller.rb

```diff
@@ -58,7 +58,7 @@ def self.modified_field
     config.add_facet_field solr_name("medium", :facetable), label: "Medium", limit: 5
     config.add_facet_field solr_name("language", :facetable), label: "Language", limit: 5
     config.add_facet_field solr_name("division", :facetable), label: "Division", limit: 5
-    config.add_facet_field solr_name("rights", :facetable), label: "Rights", limit: 5
+    config.add_facet_field solr_name("rights", :facetable), helper_method: :license_label, label: "Rights", limit: 5
```

Comment:
Awesome, sounds good, will merge. 

Yeah, `Rails.cache` is super useful for caching arbitrary expensive things. Hopefully `questioning_authority` caches the file read, instead of doing it each time you make the call, but if not that's a problem for the gem... oh, and it looks like it's actually a ruby class rather than a textual config file, so that'd happen automatically? Anyway, enough in the weeds on this one!

We should prob go ahead and switch Rails.cache to use redis at some point, just as a good practice. But not part of this PR.  (The `Rails.cache` backend is also used for fragment caching if we ever use it)